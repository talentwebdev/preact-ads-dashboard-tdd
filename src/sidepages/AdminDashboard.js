import React from 'react'
import { StatisticsList }  from '../helpers/StatisticsList';
import StatisticsItem from '../components/StatisticsItem';
function AdminDashboard() {
    return (
        <div className="adminMainBody">
            {StatisticsList.map((statisticsItem, key) =>{
                return (
                    <StatisticsItem
                        key = {key}
                        title={statisticsItem.title}
                        subTitle1={statisticsItem.subTitle1}
                        subTitle2={statisticsItem.subTitle2}
                        value1={statisticsItem.value1} 
                        value2={statisticsItem.value2}
                    />
                )
            })}
            {/* <div className="items">Item 1</div>
            <div className="items">Item 2</div>
            <div className="items">Item 3</div> */}
            <div className="items">Item 4</div>
            <div className="items">Item 5</div>
            <div className="items">Item 6</div>
        </div>
    )
}

export default AdminDashboard;
