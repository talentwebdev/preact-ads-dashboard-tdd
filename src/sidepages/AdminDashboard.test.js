import { shallow } from "enzyme";
import AdminDashboard from "./AdminDashboard";

describe("AdminDashboard", () => {
  let adminDashboardContainer;

  beforeEach(() => (adminDashboardContainer = shallow(<AdminDashboard />)));

  it("should render the div element", () => {
    expect(adminDashboardContainer.find("div").length).toBeGreaterThanOrEqual(
      1
    );
  });
  it("renders AdminDashboard correctly", () => {
    expect(adminDashboardContainer).toMatchSnapshot();
  });
});
