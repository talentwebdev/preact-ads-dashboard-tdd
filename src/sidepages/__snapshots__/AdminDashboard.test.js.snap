// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDashboard renders AdminDashboard correctly 1`] = `ShallowWrapper {}`;
